include/sequence.h

```diff
@@ -3,132 +3,131 @@
-#define NA_BGM_END_CREDITS_1 0x74           // The End/Credits I "END_DEMO"
-#define NA_BGM_OPENING_LOOP 0x75            // Forest Ambush (?)		Duplicate.
-#define NA_BGM_TITLE_THEME 0x76             // Title Screen
-#define NA_BGM_DUNGEON_APPEAR 0x77          // Surfacing of Woodfall
-#define NA_BGM_WOODFALL_CLEAR 0x78          // Woodfall Clear "PURIFICATION_0"
-#define NA_BGM_SNOWHEAD_CLEAR 0x79          // Snowhead Clear "PURIFICATION_1"
-#define NA_BGM_INTO_THE_MOON 0x7B           // To the Moon
-#define NA_BGM_GOODBYE_GIANT 0x7C           //
-#define NA_BGM_TATL_AND_TAEL 0x7D           // Tatl and Tael "CHAT"
-#define NA_BGM_MOONS_DESTRUCTION 0x7E       // Moon's Destruction "MOON_VANISH"
-#define NA_BGM_NO_MUSIC 0x7F                //
-#define NA_BGM_END_CREDITS_2 0x82           // The End/Credits II "STAFFROLL2"
+#define NA_BGM_TERMINA_FIELD 0x02           // Termina Field ("FIELD")
+#define NA_BGM_CHASE 0x03                   // Forest Chase ("CHASE")
+#define NA_BGM_MAJORAS_THEME 0x04           // Majora's Theme ("STALKID")
+#define NA_BGM_CLOCK_TOWER 0x05             // The Clock Tower ("CLOCK_TOWER")
+#define NA_BGM_STONE_TOWER_TEMPLE 0x06      // Stone Tower Temple ("RUIN_DUNGEON_FRONT")
+#define NA_BGM_INV_STONE_TOWER_TEMPLE 0x07  // Stone Tower Temple Inverted ("RUIN_DUNGEON_REVERSE")
+#define NA_BGM_FAILURE_0 0x08               // Missed Event 1 ("FAILURE_0")
+#define NA_BGM_FAILURE_1 0x09               // Failed Milk Run ("FAILURE_1")
+#define NA_BGM_HAPPY_MASK_SALESMAN 0x0A     // Happy Mask Salesman ("OMENYA")
+#define NA_BGM_SONG_OF_HEALING 0x0B         // Song of Healing ("DEMO_CURE")
+#define NA_BGM_SWAMP_REGION 0x0C            // Southern Swamp and Woodfall ("DAMP_AREA")
+#define NA_BGM_ALIEN_INVASION 0x0D          // Ghost Attack ("UFO")
+#define NA_BGM_SWAMP_CRUISE 0x0E            // Swamp Cruise ("JUNGLE_CREWS")
+#define NA_BGM_SHARPS_CURSE 0x0F            // Sharp's Curse ("SHARP")
+#define NA_BGM_GREAT_BAY_REGION 0x10        // Great Bay Coast ("SEA_AREA")
+#define NA_BGM_IKANA_REGION 0x11            // Ikana Canyon ("IKANA_AREA")
+#define NA_BGM_DEKU_PALACE 0x12             // Deku Palace ("DEKU_KING")
+#define NA_BGM_MOUNTAIN_REGION 0x13         // Mountain Village ("SNOW_AREA")
+#define NA_BGM_PIRATES_FORTRESS 0x14        // Pirates' Fortress ("PIRATE_AREA")
+#define NA_BGM_CLOCK_TOWN_DAY_1 0x15        // Clock Town Day 1. This song's filename is dummied out in MM3D ("DUMMY_21")
+#define NA_BGM_CLOCK_TOWN_DAY_2 0x16        // Clock Town Day 2. This song's filename is dummied out in MM3D ("DUMMY_22")
+#define NA_BGM_CLOCK_TOWN_DAY_3 0x17        // Clock Town Day 3. This song's filename is dummied out in MM3D ("DUMMY_23")
+#define NA_BGM_FILE_SELECT 0x18             // File Select ("FILE_SELECT")
+#define NA_BGM_CLEAR_EVENT 0x19             // Event Clear ("EVENT_CLEAR")
+#define NA_BGM_ENEMY 0x1A                   // Battle ("ENEMY")
+#define NA_BGM_BOSS 0x1B                    // Boss Battle ("BOSS00")
```

Comment:
```suggestion
#define NA_BGM_BOSS 0x1B                    // Main Dungeon Boss Battle ("BOSS00")
```
As this doesn't play for minibosses or the final boss

Reply:
I really don't see this as necessary. The sequences for the mini boss theme and for Majora's three themes are rather obviously named, so they're not hard to find. And this isn't like OoT, where there are actually two boss themes used for "normal" bosses; there's just one, and all of the four main bosses use it.